bucket/dia.json

```diff
@@ -0,0 +1,29 @@
+{
+    "homepage": "http://dia-installer.de/",
+    "license": "GPL-2.0",
+    "version": "0.97.2",
+    "description": "Drawing software",
+    "url": "http://sourceforge.net/projects/dia-installer/files/dia/0.97.2/dia_0.97.2_win32.zip/download",
+    "hash": "md5:b985954043b2c0a4664a914de24e9dc7",
+    "extract_dir": "dia",
+    "installer": {
+       "script": [
+         "Expand-7zipArchive \"$dir\\download\" \"$dir\"",
+         "Remove-Item \"$dir\\download\" -Force -Recurse"
+       ]
+    },
```

Comment:
Remove whole installer

Reply:
I guess the installer is not required anymore because with your download link, the file is directly a .zip file and scoop manage it properly ? (sorry for dumb question but I'm quite new in the scoop world :D)

Reply:
Yes